org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/manifest.xml

```diff
@@ -1,13 +1,17 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8"?>
 <manifest>
-    <!-- associated voice handler -->
-    <voice id="org.OpenT2T.Voice.Cortana.Vcd.Sample.SuperPopular.LightControl" />
-    <!-- associated schema -->
-    <schema id="org.opent2t.sample.binaryswitch.superpopular" />
-    <!-- associated onboarding module -->
-    <onboarding id="org.opent2t.onboarding.winkhub">
-        <arg name="name" value="Wink Binary Switch" />
-        <!-- when WINK devices are enumerated, the id Key contains information about the type of device. -->
-        <arg name="idKeyFilter" value="binary_switch_id" />
-    </onboarding>
-</manifest>
+  <!-- associated voice handler -->
+  <voice id="org.opent2t.sample.lightcontrol.cortana.vcd.superpopular" />
+  <!-- associated schemas -->
+  <schemas>
+      <schema id="org.opent2t.sample.binaryswitch.superpopular" main="true" type="OCF" />
+      <schema id="oic.core" type="OCF" />
+      <schema id="oic.baseResource" type="OCF" />
+  </schemas>

```

Comment:
Similarly, I think the manifest.xml for the Nest thermostat translator needs to be updated to have the same associated schemas as the Wink thermostat translator.